packages/react-cosmos-component-playground/src/components/DisplayScreen/__tests__/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Loader } from 'react-cosmos-loader';
+import DisplayScreen from '../';
+import hiThereFixture from '../__fixtures__/hi-there';
+
+const shallowLoader = element =>
+  shallow(element)
+    .dive() // Loader
+    .dive(); // PropsProxy
+
+describe('DisplayScreen', () => {
+  let wrapper;
+
+  beforeAll(() => {
+    wrapper = shallowLoader(
+      <Loader component={DisplayScreen} fixture={hiThereFixture} />
```

Comment:
We're using the Loader as a test util here. And we're importing the compiled `lib`, not the source. `Loader` is similar to `ComponentTree` back in the day. It allows us to reuse fixtures as test setup. This is a simple scenario indeed but I wanted to use it consistently in all Component Playground tests to showcase its use internally ("eating our own dog food"). It becomes more useful when we use something like a Redux fixture to create a complex test setup in one line.